scripts/images/rancher-images.txt

```diff
@@ -1,12 +1,13 @@
-docker.io/rancher/fleet-agent:v0.3.6
-docker.io/rancher/fleet:v0.3.6
+docker.io/rancher/fleet-agent:v0.3.7-rc3
+docker.io/rancher/fleet:v0.3.7-rc3
 docker.io/rancher/gitjob:v0.1.21
 docker.io/rancher/kubectl:v1.20.2
-docker.io/rancher/rancher:v2.6.0
-docker.io/rancher/rancher-webhook:v0.2.0
+docker.io/rancher/rancher:v2.6.1-rc5
+docker.io/rancher/rancher-webhook:v0.2.1
 docker.io/rancher/shell:v0.1.10
-docker.io/rancher/system-agent:v0.1.0-suc
-docker.io/rancher/system-upgrade-controller:v0.7.5
+docker.io/rancher/shell:v0.1.8
+docker.io/rancher/system-agent:v0.1.1-rc1-suc
+docker.io/rancher/system-upgrade-controller:v0.8.0-rc.1
```

Comment:
```suggestion
docker.io/rancher/system-upgrade-controller:v0.7.5
```
we need to temporarily use the old version until https://github.com/rancher/rancher/issues/34964 is resolved.

Reply:
I think we can tolerate the air-gap image for the master build if the OS upgrader needs this newer image. @bk201